### ISSUE TITLE
Allow deleting a team / a project from the admin panel

### ISSUE THREAD

**Issue by jankeromnes** [gitpod-io/gitpod#8523]
It would be useful to be able to delete a team or a project from the admin panel.

Deleting a team should also chain-delete every owned resource (like projects).

---

To illustrate one use case: There are currently 128 undeleted teams without any members in our DB:

```
mysql> SELECT id, name FROM d_b_team where id not in (select distinct teamId from d_b_team_membership) and markedDeleted = 0;
```

These member-less teams also have some projects associated to them, thus preventing these projects from being added by other users.

To "free up" these projects, it would be useful to be able to click on "Delete" on a member-less team in the admin panel.

**Comment by gtsiolis** [comment]
Yes, please! See also https://github.com/gitpod-io/gitpod/issues/6311. Cc @jldec

**Comment by gtsiolis** [comment]
From design perspective re-using the top-right corner actions to delete a team makes sense.

| User | Team (BEFORE) | Team (AFTER) |
|-|-|-|
| ![admin-user](https://user-images.githubusercontent.com/120486/156351442-631cd220-9ec9-4102-941f-14f8f5ba1627.png) | ![admin-team-before](https://user-images.githubusercontent.com/120486/156351450-e62e82ed-034e-4166-b440-a6484d7f3f65.png) | ![admin-team-after](https://user-images.githubusercontent.com/120486/156351456-d9c4ec2e-c8a9-4fb1-8cd4-017dabc9ff52.png) |

**Comment by gtsiolis** [comment]
Reopening as this is still relevant. Cc @jldec

**Comment by jimmybrancaccio** [comment]
This would have been useful to have as a customer submitted a ticket in Front (`cnv_jjlc2bu`) where there was no one on the team and no way to delete the Team from the admin UI.
